install.sh

```diff
@@ -0,0 +1,117 @@
+export DEBIAN_FRONTEND=noninteractive
+
+echo '>> Updating information about packages...'
+sudo sh -c 'echo "deb http://cran.rstudio.com/bin/linux/ubuntu trusty/" >> /etc/apt/sources.list' # R
+gpg --keyserver keyserver.ubuntu.com --recv-key E084DAB9
+gpg -a --export E084DAB9 | sudo apt-key add -
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ wily-pgdg main" >> /etc/apt/sources.list.d/pgdg.list' # PostgreSQL
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+sudo apt-get -y update
+sudo apt-get -y install vim
+
+echo '>> Installing xfce4 and VirtualBox guest tools...'
+sudo apt-get -y install xfce4 virtualbox-guest-dkms virtualbox-guest-utils virtualbox-guest-x11
+sudo sh -c 'echo "allowed_users=anybody" > /etc/X11/Xwrapper.config'
+
+echo '>> Installing pip...'
+wget --quiet https://bootstrap.pypa.io/get-pip.py
+sudo apt-get -y install python
+sudo python get-pip.py
+sudo apt-get -y install python-dev build-essential
+
+echo '>> Installing virtualenv...'
+sudo pip install -U virtualenv==15.0.3
+mkdir ~/.virtualenvs
+sudo pip install -U virtualenvwrapper==4.7.2
+sh -c 'echo "\nexport WORKON_HOME=~/.virtualenvs\nsource /usr/local/bin/virtualenvwrapper.sh" >> .bash_profile'
+source .bash_profile
+
+echo '>> Installing dependencies for ReproZip...'
+sudo apt-get -y install gcc libsqlite3-dev libyaml-dev
+sudo pip install -U reprozip
+
+echo '>> Installing git...'
+sudo apt-get -y install git
+
+echo '>> Installing dependencies for digits-sklearn...'
+mkvirtualenv --system-site-packages digits-sklearn
+cd reprozip-examples/digits-sklearn/
+pip install -r requirements.txt
+cd ../
+deactivate
+
+echo '>> Installing dependencies for digits-sklearn-opencv...'
+mkvirtualenv --system-site-packages digits-sklearn-opencv
+cd digits-sklearn-opencv/
+pip install -r requirements.txt
+cd ../../
+sudo apt-get -y install cmake pkg-config
+sudo apt-get -y install libjpeg8-dev libtiff5-dev libjasper-dev libpng12-dev
+sudo apt-get -y install libgtk2.0-dev
+sudo apt-get -y install libatlas-base-dev gfortran
+git clone https://github.com/opencv/opencv.git
```

Comment:
I don't know any other ways to install OpenCV -- does it come with the Ubuntu distribution? This was the standard compilation / installation step I found in many forums on the Web for Ubuntu. Let me know if there is any other way to install OpenCV.
Also, I did not have to chance any of the code for OpenCV 3.0.0, but you are right that it is better to use the same one from the RPZ package (I did not even realize that), so no issued on that.